project/profiles/capacity4more/themes/c4m/kapablo/templates/ds-1col--node-group-dashboard.tpl.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * @file
+ * Display Suite 1 column template.
+ */
+?>
+<<?php print $ds_content_wrapper; print $layout_attributes; ?> class="ds-1col <?php print $classes; ?> clearfix">
+
+<?php if (isset($title_suffix['contextual_links'])): ?>
+  <?php print render($title_suffix['contextual_links']); ?>
+<?php endif; ?>
+
+<div class="content clearfix group-details">
```

Comment:
@nirgn975 
It seems that the only diff between this template and the original is that in here we add the `group-details` class, but I think we can just add the class via the configuration of the view, no ?